src/app/[locale]/components/Header.tsx

```diff
@@ -23,22 +28,24 @@ const Header = ({ homepage, logo }: HeaderProps) => {
   return (
     <>
       <PageHeader>
-        {homepage ? (
-          <Link
-            boxContent
-            href={homepage.href}
-            aria-labelledby="logo-link-label"
-          >
-            <span id="logo-link-label" hidden>
-              {homepage.label}
-            </span>
-            {logoElement}
-          </Link>
-        ) : (
-          logoElement
-        )}
+        <div className="flex flex-row items-center justify-between">
+          {homepage ? (
+            <Link
+              boxContent
+              href={homepage.href}
+              aria-labelledby="logo-link-label"
+            >
+              <span id="logo-link-label" hidden>
+                {homepage.label}
+              </span>
+              {logoElement}
+            </Link>
+          ) : (
+            logoElement
+          )}
+          {config && config.base.multilanguage && <LanguageSwitch />}
```

Comment:
Als we dan via supportedLanguages zouden werken kunnen we hier kijken of er meer dan 1 aanwezig is